### PR TITLE
Test type info

### PR DESCRIPTION
**Pull request by JonasVautherin**
Loading the file statically is not nice for unit testing. But loading the file for each instantiation doesn't sound right either. So I made a factory for that, that loads the file only once.

### CODE REVIEW

File: pb_plugins/dcsdkgen/type_info.py [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
 
+class TypeInfoFactory:
+
+    def __init__(self):
+        self._conversion_dict = self.__load_conversions_dict()
+
+    def create(self, field):
+        return TypeInfo(field, self._conversion_dict)
+
+    def __load_conversions_dict(self):
```
Comment (xvzf) [inline pb_plugins/dcsdkgen/type_info.py]: `__` is usually an python internal function name, I'd suggest to go with just one `_`!

File: pb_plugins/dcsdkgen/type_info.py [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+        try:
+            _conversion_dict_path = environ.get(
```
Comment (xvzf) [inline pb_plugins/dcsdkgen/type_info.py]: Could be more readable when using an formatted String (`f"this is a text with {variable_name_or_function}"`

File: pb_plugins/dcsdkgen/type_info.py (hunk 1 above)
Comment (JonasVautherin) [reply pb_plugins/dcsdkgen/type_info.py]: `_` is private, and `__` internal then?

Comment (xvzf) [reply pb_plugins/dcsdkgen/type_info.py]: Exactly! There is no such thing as private in python but it is the common way to do it over here :)

**Comment by JonasVautherin** [comment]
@xvzf Should be better now! And done!

**Pull request merged**